src/less-style/question-props.less

```diff
@@ -157,3 +157,17 @@
   padding: 0 2em;
   margin-bottom: 2em;
 }
+
+.fake-textarea {
+    -moz-appearance: textfield-multiline;
+    -webkit-appearance: textarea;
+    border: 1px solid #cccccc;
+    border-radius: 4px;
+    font: medium -moz-fixed;
+    font: -webkit-small-control;
+    height: 28px;
+    overflow: auto;
+    padding: 2px;
+    resize: both;
+    width: 400px;
```

Comment:
Setting all pseudo-textareas to a pixel width seems suspicious. Is this temporary?